apps/src/StudioApp.js

```diff
@@ -1934,22 +1934,12 @@ StudioApp.prototype.handleEditCode_ = function (config) {
     }
   }
 
-  // Remove maker API blocks from palette and autocomplete, unless maker APIs are enabled.
-  // We didn't have access to project.useMakerAPIs() when dropletConfig
-  // was initialized, so include it initially, and conditionally remove it here.
+  // Remove maker API blocks from palette, unless maker APIs are enabled.
   if (!project.useMakerAPIs()) {
-    //// Remove maker blocks from the palette
+    // Remove maker blocks from the palette
```

Comment:
this seems like it shouldn't be necessary given that `makeDisabledConfig` "removes all blocks from categories". If that's not true, maybe you can clarify what the difference is?